frontend/src/util/activity.js

```diff
@@ -79,15 +87,17 @@ const getActivityParams = (message) => {
   return null
 }
 
-const createActivity = async (activityParams, activityMessageList, apiUrl) => {
+const createActivity = (activityParams, activityMessageList, apiUrl) => {
   switch (activityParams.entityType) {
     case TLM_ET.CONTENT:
-      return await createContentActivity(activityParams, activityMessageList, apiUrl)
+      return createContentActivity(activityParams, activityMessageList, apiUrl)
```

Comment:
I understand it works but am not fond of having a function that sometimes returns a value and other times a promise that  needs to be awaited. It also makes implicit the fact that `await`/`.tĥen()` is required to get the value.